test/application/respond.js

```diff
@@ -544,7 +544,6 @@ describe('app.respond', () => {
           ctx.length = fs.readFileSync('package.json').length;
           const stream = fs.createReadStream('package.json');
           ctx.body = stream;
-          ctx.body = stream;
```

Comment:
from the description of this test case(should keep content-length if overwritten with the same stream), I think we should not delete this line here.

Reply:
right, sorry I missed the case description